requirements.txt

```diff
@@ -1,10 +1,9 @@
-molo.core>=10.0.0,<10.2.0
+molo.core>=11.0.0,<12.0.0
 # molo.forms.post_comment is a direct copy of django_comments.views.comments.post_comment
 # added this to add form kwargs, upgrade with care
-django-contrib-comments==1.9.1
-django-mptt==0.10.0
```

Comment:
Why are you removing this library? Isn't this required to do tree-style comment replies?

Reply:
it is installed with molo